src/lib/errors.js

```diff
@@ -38,6 +38,10 @@ export function checkForCommonErrors() {
         $logger.warn(`ie_intranet_mode`);
     }
 
+    if (isIE() && !isIECompHeader()) {
+        $logger.warn(`Drop tag: <meta http-equiv="X-UA-Compatible" content="IE=edge">`);
```

Comment:
Can we make this an alphanumeric event, since it ends up in our logging back-end?

Maybe `$logger.warn(`ie_meta_compatibility_header_missing`, { message: `Drop tag: <meta http-equiv="X-UA-Compatible" content="IE=edge">` });`?